src/server/pfs/s3/auth.go

```diff
@@ -3,32 +3,45 @@ package s3
 import (
 	"net/http"
 
+	"github.com/gorilla/mux"
 	"github.com/pachyderm/pachyderm/src/client/auth"
 	"github.com/pachyderm/pachyderm/src/client/pkg/errors"
 )
 
 func (c *controller) SecretKey(r *http.Request, accessKey string, region *string) (*string, error) {
-	pc, err := c.clientFactory.Client(accessKey)
+	c.logger.Debugf("SecretKey: accessKey=%+v, region=%+v", accessKey, region)
```

Comment:
Just checking: is `accessKey` secret? Are there risks associated with logging it?

Reply:
Yeah it is, good point. I'll remove this.